src/plot.py

```diff
@@ -78,3 +87,54 @@ def plot_proportions(
         )
 
     plt.show()
+
+
+def plot_training_curves(history, training_dir):
+    """
+    Saves plots of the training curves (loss and other metrics) based on the
+    training history.
+
+    Args:
+        history (History): History object returned by `model.fit()`.
+        training_dir (str): Path to the directory where the plot will be saved.
+    """
+    # Create plot for loss and other metrics
+    plt.figure(figsize=(10, 5))
+
+    # Plot for loss
+    plt.plot(history.history["loss"], label="Loss (training)")
+    if "val_loss" in history.history:
+        plt.plot(history.history["val_loss"], label="Loss (validation)")
+
+    # Add title, legend, and axis labels
+    plt.title("Loss Curves")
+    plt.xlabel("Epoch")
+    plt.ylabel("Loss")
+    plt.legend()
+
+    # Save the plot as a PNG file
```

Comment:
Close in a function, do not repeat code for accuracy, make it work for other metrics too